Pilot/pilotTools.py

```diff
@@ -271,25 +271,15 @@ def getSubmitterInfo(ceName):
         flavour = "SSH%s" % batchSystemType
         pilotReference = "sshslurm://" + ceName + "/" + batchSystemJobID
 
-    # Condor
-    if "CONDOR_JOBID" in os.environ:
+    # HTCondor
+    if "_CONDOR_JOB_AD" in os.environ:
         batchSystemType = "HTCondor"
-        batchSystemJobID = os.environ["CONDOR_JOBID"]
-        batchSystemParameters["InfoPath"] = os.environ.get("_CONDOR_JOB_AD", "Unknown")
+        batchSystemJobID = None # Not available in the environment
+        batchSystemParameters["InfoPath"] = os.environ["_CONDOR_JOB_AD"]
 
         flavour = "SSH%s" % batchSystemType
         pilotReference = "sshcondor://" + ceName + "/" + batchSystemJobID
```

Comment:
Same problem here, `batchSystemJobID` would be `None`

Reply:
Oops sorry, I should have taken more time to read the code I wrote :sweat_smile: 
So I came up with a mixed solution, namely: if `CONDOR_JOBID` is defined (it seems to be the case when SSH+Condor is used), use it. Else use the pilot uuid.
I made the choice of keeping `sshcondor://ceName` as part of the pilot reference because it provides details about the CE and the Batch System used, but if you feel that only the pilot uuid should be used if `CONDOR_JOBID` is not set, I can modify the code.